### ISSUE TITLE
[cwm] Customize the web map to fit our demo app

### ISSUE THREAD

**Issue by dadiorchen** [Greenstand/treetracker-web-map-client#1686]
We are building our demo app to demonstrate the integration of the map, this app gets its own UI design and color, style, we need to customize the web map client to fit it.

https://www.figma.com/file/b9ysoeaLAFcjy2Dlrbu5Ya/Web-Map-Demo?type=design&node-id=456-4088&mode=design&t=g3TqVXF15IkDMB86-4

https://www.figma.com/file/b9ysoeaLAFcjy2Dlrbu5Ya/Web-Map-Demo?type=design&node-id=429-2925&mode=design&t=g3TqVXF15IkDMB86-0

<img width="475" alt="image" src="https://github.com/Greenstand/treetracker-web-map-client/assets/5744708/720794cc-0b51-4b1d-afec-03df25e1f896">

---

Some hints:

- Please read our [readme](https://github.com/Greenstand/treetracker-web-map-client#treetracker-web) for more information/guide/tutorial.
- For design guidelines checkout our [Figma design](https://www.figma.com/file/XdYFdjlsHvxehlrkPVYq0l/Greenstand-Webmap).
- Here is [an engineering book](https://greenstand.gitbook.io/engineering/) in Greenstand.
- To know more about our organization, visit our [website](https://greenstand.org).
- If you want to join the slack community (some resources need the community member's permission), please leave your email address.

**Comment by dadiorchen** [comment]
@RubenSmn do you think this is possible?

**Comment by RubenSmn** [comment]
Yes @dadiorchen the theme editor should be able to handle all the changes that are necessary to achieve this UI, the only thing that we're not able to change yet is the spacing.

**Comment by dadiorchen** [comment]
@RubenSmn please also consider this use case, here, the wallet demo, it is more like a concept of `client` than `user` (like organization), which means, we custom the theme for this app/client, no matter which user, org, wallet is showing. We need to define this kind of use case.
